README.md

```diff
@@ -1,320 +1,66 @@
-
-```
-celestia-appd keys add --recover <keyname>
-```
-
-followed by a prompt to enter a bip39 mnemonic, which is the mnemonic that was created as part of `1_create_key.sh` script in the first step.
-
-You'll also be asked for a passphrase which is an input you have to define.
-
-After this fix repeat step 6.
+- [Run Celestia Devnet](#run-celestia-devnet)
+  - [Celestia 101(Overview)](#celestia-101overview)
+    - [Celestia Full Node](#celestia-full-node)
+    - [Celestia Light Client](#celestia-light-client)
+  - [Installation guide](#installation-guide)
+  - [Troubleshoot](#troubleshoot)
 
+## Celestia 101(Overview)
+<i>Conceptually</i>, we have 2 main components in the network: 
+1. `Celestia Full Node`.
+2. `Celestia Light Client`.
 
-## Running a Celestia Node
-### Pre-Requisites
-You need to have the trusted hash in order to initialize the full celestia node
-In order to know the hash, you need to query the celestia-app:
-```sh
-curl -s http://localhost:26657/block?height=1 | grep -A1 block_id | grep hash
-```
+> Note: When you see formated text of those 2 above(`Celestia Full Node`, `Celestia Light Client`), remember that we are refering to a <i>conceptual</i> view. Standard formated text is a technical view 
 
-### Full Node Configuration
-1. Initialize the full node
-```sh
-celestia full init --core.remote <ip:port of celestia-app> --headers.trusted-hash <hash_from_celestia_app>
-```
+### Celestia Full Node
+When we are mentioning `Celestia Full Node`, we are relating to a combination of 2 components: 
 
-Example:
-```sh 
-celestia full init --core.remote tcp://127.0.0.1:26657 --headers.trusted-hash 3BDFBD7E2D97215CFA600DD8B39AAEECC015E43FEE7B8A4D8A8B630E8B4D4006
-```
+- Celestia Application (lives in [celestia-app](https://github.com/celestiaorg/celestia-app) repo)
+- Celestia Full Node (lives in [celestia-node](https://github.com/celestiaorg/celestia-node) repo)
 
-2. Edit configurations
+<i>Conceptually</i>, one can't live without another as we need every new block to be erasure coded in order to do data availability sampling from `Celestia Light Clients`
 
-If you have multiple celestia full nodes, then you need to add them as mutual peers in the `config.toml` file and allow the peer exchange. This is needed in order for celestia full nodes communication between each other.
-```sh
-nano ~/.celestia-full/config.toml
-```
-```sh
-...
-[P2P]
-  ...
-  MutualPeers = ["/ip4/<ip>/tcp/2121/p2p/<pubKey>"] #add multiaddresses of other celestia full nodes
-  PeerExchange = true #change this line to true. Be default it's false
-  ...
-...
-```
+Technically, Celestia Application and Celestia Full Node are operating on different processes of <b>one instance</b>(e.g. server). Let's break down what each of the parties do. 
 
-3. Start the full node
-```sh
-celestia full start
-```
-Now, the celestia full node will start syncing headers and storing blocks from celestia app. 
+Celestia Application(CA) takes care of these points: 
```

Comment:
I'd prefer we use "cApp" or "Capp" or "CApp" here as it's makes it a bit more clear what we're talking about. Ignoring the unfortunate similarity to a "dApp", but that's Cosmos's fault for further overloading the term "application"

Reply:
c-app or cApp or CApp is good. Celestia Core will be run in tandem with Celestia App anyways. I think a quick diagram would help a lot IMO.

Reply:
True. I'll add the line for Celestia Core in tandem with Celestia Application as well as changing the abbreviation from CA to C-APP